managed-services/webhosting/how-to/order-webhosting.mdx

```diff
@@ -25,24 +25,25 @@ This page shows how to create a Scaleway Web Hosting plan. Web Hosting is a mana
 </Message>
 
 1. Click **Web Hosting** in the **Managed Services** section of the side menu. The Web Hosting splash page displays.
-
-
     <Message type="note">
       If you have existing Web Hosting plans in your account, you will see a list of
       these. Click the <Icon name="plus" />
       button to create additional Web Hosting plans.
     </Message>
 2. Click **Order Web Hosting**. The Web Hosting creation wizard displays.
+3. Choose a **region** and a **control panel** for your Web Hosting. Then click **Configure Web Hosting**.
 3. Complete the following steps in the wizard: 
-    - Enter a **domain name** to link to your webhosting account. 
+    - Enter a **domain name** to link to your web hosting account. 
       <Message type="important">
         The domain must be registered already and you have to be the registrant for the domain. If you do not have a domain name yet, you should [register a domain name](/network/domains-and-dns/how-to/register-internal-domain/) before continuing.
       </Message>
     - Choose a **hosting plan**. Choose the offer that corresponds to the requirements for your Web Hosting plan. 
-    - Choose wether you want to enable the **backup option** or not. 
-      <Message type="tip">
-        Activate this option to benefit from a daily automated backup of your webhosting account, accessible by FTP. The offer includes a daily backup of your website, emails and databases, retention of these backups for three days, and retention of a full weekly backup of your website for two months.
-      </Message>
+    - Configure the options.
+        - Choose wether you want to enable the **backup option** or not. 
+            <Message type="tip">
+            Activate this option to benefit from a daily automated backup of your web hosting account, accessible by FTP. The offer includes a daily backup of your website, emails and databases, retention of these backups for three days, and retention of a full weekly backup of your website for two months.
+            </Message>
+        - Choose if you want to obtain a dedicated IP address for your Web Hosting. A dedicated IP allows you to benefit from improved email reputation and optimal SSL setup.
```

Comment:
```suggestion
        - Choose whether you want to obtain a dedicated IP address for your web hosting. A dedicated IP allows you to benefit from improved email reputation and optimal SSL setup.
```

Reply:
Dunno about the caps, I can't tell anymore if it's a reference to the product or something generic...